resource_alks_iamtrustrole.go

```diff
@@ -67,6 +67,11 @@ func resourceAlksIamTrustRole() *schema.Resource {
 				Default:  false,
 				Optional: true,
 			},
+			"max_session_duration_in_seconds": {
+				Type:     schema.TypeInt,
+				Default:  3600,
+				Optional: true,
+			},
```

Comment:
If we're adding this field to trust roles, we should make sure that it works. Right now the `resourceAlksIamTrustRoleCreate` function doesn't read this field and pass it along in the API call to ALKS Go so it's effectively a broken field. The reason you probably had to add this was because `alks_iamtrustrole` shares the same read, update, and delete functions as the `alks_iamrole` resource, but it still has its own create function so that function needs to be updated

Reply:
Added